README.md

```diff
@@ -24,23 +24,23 @@ Neovim is a project that seeks to aggressively refactor Vim in order to:
 - Enable [advanced UIs] without modifications to the core
 - Maximize [extensibility](https://github.com/neovim/neovim/wiki/Plugin-UI-architecture)
 
-See [the wiki](https://github.com/neovim/neovim/wiki/Introduction) and [Roadmap]
+See the [Introduction](https://github.com/neovim/neovim/wiki/Introduction) wiki page and [Roadmap]
 for more information.
 
 [![Throughput Graph](https://graphs.waffle.io/neovim/neovim/throughput.svg)](https://waffle.io/neovim/neovim/metrics)
 
 Features
 --------
 
-- Modern [GUIs](https://github.com/neovim/neovim/wiki/Related-projects#gui)
+- Modern [GUI clients](https://github.com/neovim/neovim/wiki/Related-projects#gui-clients)
```

Comment:
Please don't change the anchor link on the wiki page, it's already used in many places.

Reply:
Thanks for pointing that out. I did not realize its significance.

Reply:
```suggestion
- Modern [GUIs](https://github.com/neovim/neovim/wiki/Related-projects#gui)
```